back/engines/commercial/report_builder/app/services/report_builder/queries/survey_question_result.rb

```diff
@@ -16,6 +18,12 @@ def run_query(
           group_mode: group_mode,
           group_field_id: group_field_id
         )
+      elsif year && quarter
+        Surveys::ResultsWithDateGenerator.new(
+          phase,
+          year: year,
+          quarter: quarter
+        )
```

Comment:
@amanda-anderson Currently survey widgets will only be able to group OR filter by quarter - not both

Reply:
Is it quite technically complex to support both?

Reply:
(Update from our discussion this morning: This behaviour is good for now, we'll add a ticket to our board to look into supporting both still though)